### DISCUSSION TITLE
Team being removed as a Pull Request reviewer after one member has reviewed

### DISCUSSION THREAD

**Discussion by pearsonradu** [community/community#16853] [Pull Requests] [answered]
Migrating this from https://github.community/t/team-being-removed-as-a-pull-request-reviewer-after-one-member-has-reviewed/3250.

Originally from @dgard1981:

We have applied a CODEOWNERS file to our code, which contains the name of a team, in the following format.

* @Org-Name/Team-Name
We’ve also given that team write permissions to the repository, ensuring that the team is added as a reviewer when a Pull Request is created.

When a Pull Request is created, every member of the code owning team gets an email, which is expected. However, once one member of the team has completed their review of the Pull Request, the team is removed as a reviewer and the individual is added.

This is very undesirable, as now the rest of the code owning team loses visibility of the Pull Request through [https://github.com/pulls 31](https://github.com/pulls).

Is there any way to change this behaviour?

**Comment by usmonster** [comment, marked as answer]
Duplicate of #5289, yes? Any way we can merge these?
